### DISCUSSION TITLE
Netbox v3.7.6's Interface's exported CSV can't be re-imported with the device name changed to another existing device: errors: header name case mismatch and invalid type choice

### DISCUSSION THREAD

**Discussion by LinuxMagicSupportTeam** [netbox-community/netbox#16189] [Help Wanted!]
Devices / Device Components / Interfaces / Export|Import:

Issue 1: the exported CSV's headers have upper case letters which the Import process can't recognize.
Issue 2: all the "type" values seem to be invalid choices, despite the data being specified by the Export process

sample CSV data:
```
name,device,label,enabled,type,description
Eth1,testdev,,True,SFP (2GFC),
ETH2,testdev,,True,1000BASE-T (1GE),
ETH3,testdev,,True,100BASE-FX (10/100ME FIBER),
ETH4,testdev,,True,1000BASE-T (1GE),
ETH5,testdev,,True,1000BASE-T (1GE),
SFP-1,testdev,,True,SFP (1GE),
SFP-2,testdev,,True,SFP (1GFC),
```
e.g. Error shown: ```
Record 1 type: Select a valid choice. SFP (2GFC) is not one of the available choices. 
SFP (2GFC) is not one of the available choices. 
```

**Comment by LinuxMagicSupportTeam** [comment]
apologies; posted in the wrong location; should've been in Issues:
https://github.com/netbox-community/netbox/issues/16190

**Comment by jcollie** [comment]
This is a deliberate design decision by the NetBox developers, and they are _not_ open to changing that.

https://github.com/netbox-community/netbox/issues/14502

https://github.com/netbox-community/netbox/wiki/Frequently-Asked-Questions#why-cant-i-pass-exported-data-directly-into-the-import-form
